airbyte-integrations/connectors/source-twilio/integration_tests/configured_catalog.json

```diff
@@ -0,0 +1,1855 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "accounts",
+        "json_schema": {
```

Comment:
Recently, we can set `configured_catalog` without a `json_schema`, leaving an empty object instead. Therefore, you can specify `{} `for `json_schema` for each stream.